continuous_integration/recipes/dask/meta.yaml

```diff
@@ -20,15 +20,15 @@ build:
 requirements:
   host:
     - python >=3.8
-    - setuptools
```

Comment:
Should we add `dask-core` and `distributed` as was done [in the feedstock]( https://github.com/conda-forge/dask-feedstock/blob/fa81d16d6350f91f0b6b042c3a17da2612517973/recipe/meta.yaml#L19-L20 )?

Reply:
Same reasoning here as in https://github.com/dask/distributed/pull/7201#discussion_r1006006871, happy to make a similar change if you think we should keep the host dependencies consistent

Reply:
Sure let's discuss in that thread and update both accordingly